docs/broadcastMessage.md

```diff
@@ -0,0 +1,655 @@
+# Working with Broadcast Message
+
+[ [Overview](#overview) ]  [ [Operation details](#operation-details) ]  [ [Sample configuration](#sample-configuration) ]
+
+
+## Overview
+
+The following operations allow you to broadcast a message.
+
+Validation - Maximum total JSON size of the request is 30kb. The maximum list length is 300 receivers. 
+
+Click an operation name to see details on how to use it.
+
+**Related Viber documentation**
+https://developers.viber.com/docs/api/rest-bot-api/#broadcast-message
+
+
+| Operation | Description |
+| ------------- |-------------|
+| [broadcastCarouselContentMessage](#Broadcast-carousel-content-message)|Broadcast a carousel content message.|
+| [broadcastContactMessage](#Broadcast-contact-message) | Broadcast a contact.|
+| [broadcastFileMessage](#Broadcast-file-message) | Broadcast a file.|
+| [broadcastLocationMessage](#Broadcast-location-message) |Broadcast a location.|
+| [broadcastPictureMessage](#Broadcast-picture-message) |Broadcast a picture.|
+| [broadcastStickerMessage](#Broadcast-sticker-message) | Broadcast a sticker.|
+| [broadcastTextMessage](#Broadcast-text-message) | Broadcast a text message.|
+| [broadcastUrlMessage](#Broadcast-url-message) | Broadcast a url.|
+| [broadcastVideoMessage](#Broadcast-video-message) |Broadcast a video.|
```

Comment:
You need to add Sample response and Related API doc for all the methods.
Fix in all the places